ios/wrappers/CioRctWrapper.swift

```diff
@@ -0,0 +1,117 @@
+import CioDataPipelines
+import CioMessagingInApp
+import CioMessagingPush
+import UserNotifications
+import React
+
+func flush() {
+#if DEBUG
+    CustomerIO.shared.flush()
+#endif
+}
+
+@objc(CioRctWrapper)
+class CioRctWrapper: NSObject {
+    
+    @objc var moduleRegistry: RCTModuleRegistry!
+        
+    @objc
+    func initialize(_ configJson: AnyObject, logLevel: String) {
+        do {
+            let rtcConfig = try RCTCioConfig.from(configJson)
+            let cioInitConfig = cioInitializeConfig(from: rtcConfig, logLevel: logLevel)
+            CustomerIO.initialize(withConfig: cioInitConfig.cdp)
+            if let inAppConfig = cioInitConfig.inApp {
+                    // In app initializes UIView(s) which would crash if run from non-UI queues
+                DispatchQueue.main.async {
+                    MessagingInApp.initialize(withConfig: inAppConfig)
+                    MessagingInApp.shared.setEventListener(self)
+                }
+            }
+            flush()
+        } catch {
+            // TODO: Add log when logger feature is implemented
+        }
+    }
+    
+    @objc
+    func identify(_ userId: String? = nil, traits: [String: Any]? = nil) {
+        if let userId, let traits {
+            CustomerIO.shared.identify(userId: userId, traits: traits)
+        } else if let userId {
+            CustomerIO.shared.identify(userId: userId, traits: traits)
+        } else {
+            // TODO: Add log when logger feature is implemented
+        }
```

Comment:
This could be simplify with guard statement? 🤔

Reply:
Right, I further simplified it by removing the conditional check.